.github/scripts/generate_ci_workflows.py

```diff
@@ -179,6 +180,15 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
         enable_doc_jobs=True,
         num_test_shards=2,
     ),
+    CIWorkflow(
+        arch="linux",
+        build_environment="pytorch-linux-backward-compatibility-check-test",
+        docker_image_base=f"{DOCKER_REGISTRY}/pytorch/pytorch-linux-xenial-py3.6-gcc5.4",
+        test_runner_type=LINUX_CPU_TEST_RUNNER,
+        on_pull_request=True,
+        num_test_shards=2,
```

Comment:
I don't think test_shards is handled for backward tests, can you help check the logs from both shards - I guess they are testing same things, if yes might want to set it to 1 or remove this field

Reply:
> I don't think test_shards is handled for backward tests, can you help check the logs from both shards - I guess they are testing same things, if yes might want to set it to 1 or remove this field

Pretty sure they are... I was wondering where the /0 and /1 are coming from.

Reply:
Shards does not really work for backward compat tests, so let's keep them at one

Reply:
Is an entirely new workflow really needed for this? Seems like this is just an extension of `pytorch-linux-xenial-py3.6-gcc5.4` in general

@samestep did a lot of work to enable separate configurations within that workflow

Reply:
As far as I could understand, the script for the bw compat test installs a nightly build of PyTorch anyway - https://github.com/pytorch/pytorch/blob/d6e776d96110a3b0c7d4777b03e4c400ae9a46eb/.jenkins/pytorch/test.sh#L369. Wonder if it is possible to run it against some manifestly pre-existing Docker image rather than require a fresh build that will get overwritten anyway?